### PR TITLE
fix(chat): fix attachments upload (Issue #1229)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

Fix attachments upload

Issues:

- #1229

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1290.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1290.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/utils/app/file.ts [hunk 1]
```diff
@@ -123,19 +123,19 @@ export const getFilesWithInvalidFileType = (
     ? []
     : files.filter((file) => !isAllowedMimeType(allowedFileTypes, file.type));
 };
-export const notAllowedSymbols = ':;,=/{}%&\\\t\x00-\x1F';
+export const notAllowedSymbols = ':;,=/{}%&\\';
 export const notAllowedSymbolsRegex = new RegExp(
-  `[${escapeRegExp(notAllowedSymbols)}]|(\r\n|\n|\r|\t)`,
+  `[${escapeRegExp(notAllowedSymbols)}]|(\r\n|\n|\r|\t)|[\x00-\x1F]`,
```
Comment (IlyaBondar) [inline apps/chat/src/utils/app/file.ts:128]: I suppose this change disallow `[` and `]`, Please, use just `\x00-\x1F`. At least double check it

File: apps/chat/src/components/Files/PreUploadModal.tsx [hunk 2]
```diff
@@ -409,9 +449,7 @@ export const PreUploadDialog = ({
           onClick={handleUpload}
           disabled={selectedFiles.length === 0}
         >
-          {customUploadButtonLabel
-            ? customUploadButtonLabel
-            : t('Upload and attach files')}
+          {customUploadButtonLabel ? customUploadButtonLabel : t('Upload')}
```
Comment (IlyaBondar) [inline apps/chat/src/components/Files/PreUploadModal.tsx:452]: Maybe need to `Upload and attach` instead?

Comment (Alexander-Kezik) [reply apps/chat/src/components/Files/PreUploadModal.tsx:452]: On a daily meeting it was decided to leave 'upload' here

File: apps/chat/src/utils/app/file.ts (hunk 1 above)
Comment (Alexander-Kezik) [reply apps/chat/src/utils/app/file.ts:128]: https://javascript.info/regexp-character-sets-and-ranges#ranges
that's a way to exclude a range
I still can use square brackets 
Screenshot 2024-04-23 at 11 09 35

**Review by IlyaBondar — DISMISSED** [review]

**Review by IlyaBondar — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1290.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1290.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1290.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**